bigtable-hbase/src/main/resources/com/google/cloud/bigtable/hbase/bigtable-hbase.properties

```diff
@@ -0,0 +1 @@
+project.version=${project.version}
```

Comment:
hm... the properties file belongs to com/google/cloud/bigtable/hbase package, do we still need to specify bigtable.hbase.version?
If we want to keep track of netty version and grpc version, maybe we should use different properties, could be in the same properties file.